### PR TITLE
List implementation using STL #29

### PR DESCRIPTION
**Pull request by frost-queen**
forward list and list implemented in seperate files.

### CODE REVIEW

**Comment by frost-queen** [comment]
okay working on it

**Comment by frost-queen** [comment]
@ramitsawhney27  Kindly review the changes.

File: Data Structures/Linked-Lists/forwardlist_using_stl.cpp [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#include<iostream>
+#include<vector>
+#include<forward_list>/*header file required for forward list implementation*/
+using namespace std;
+//display functions
+void displayint(forward_list <int> f)
```
Comment (aditya81070) [inline Data Structures/Linked-Lists/forwardlist_using_stl.cpp]: Follow [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards#naming-convention) everywhere in your code.

**Comment by aditya81070** [comment]
@frost-queen Please make required changes/

**Comment by frost-queen** [comment]
Okay will make these changes

**Comment by frost-queen** [comment]
Done with recommended changes. @aditya81070

**Comment by sahilbansal17** [comment]
@ramitsawhney27 @aditya81070 Shall we merge this PR ?

**Pull request merged**